app/src/main/java/org/gdg/frisbee/android/onboarding/Invite.java

```diff
@@ -0,0 +1,38 @@
+package org.gdg.frisbee.android.onboarding;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+class Invite implements Parcelable {
```

Comment:
Why do you add this class? Is this "just" for giving String a meaning? Then `sender` should be `gplusIdOfSender` or so